src/res/package.template.json

```diff
@@ -18,8 +18,8 @@
     "webgme": "<%= webgmeVersion %>"
   },
   "devDependencies": {
-    "mocha": "^2.2.5",
-    "rimraf": "^2.4.0",
-    "chai": "^3.0.0"
+    "mocha": "^5.2.0",
+    "rimraf": "^2.6.2",
```

Comment:
On second thought.. is rimraf even needed? Chai might not be needed either (the generated tests, at least for plugins obtains these from the testFixture).

Reply:
It doesn't look like it. Good catch